.github/actions/create-pr/action.yml

```diff
@@ -0,0 +1,92 @@
+name: "Create PR custom action"
+description: "Create a PR and a temporary branch, close duplicates"
+
+# PROCESS
+#
+# 1. Setup git client using Powertools for AWS Lambda bot username
+# 2. Pushes staged files to a temporary branch
+# 3. Creates a PR from temporary branch against a target branch (typically trunk: develop, main, etc.)
+# 4. Searches for duplicate PRs with the same title
+# 5. If duplicates are found, link to the most recent one, close and delete their branches so we keep a single PR
+# 6. In the event of failure, we delete the now orphaned branch (if any), and propagate the failure
+
+# REQUIREMENTS
+# You need to enable "Allow GitHub Actions to create and approve pull requests" in your repository settings
+# You need to have "skip-changelog" label in your repository to skip changelog generation.
+
+# USAGE
+#
+# - name: Create PR
+#   id: create-pr
+#   uses: ./.github/actions/create-pr
+#   with:
+#     files: "CHANGELOG.md"
+#     temp_branch_prefix: "ci-changelog"
+#     pull_request_title: "chore(ci): changelog rebuild"
+#     github_token: ${{ secrets.GITHUB_TOKEN }}
+# - name: Step to demonstrate how to access outputs (no need for this)
+#   run: |
+#     echo "PR number: ${PR_ID}"
+#     echo "Branch: ${BRANCH}"
+#   env:
+#     PR_ID: ${{ steps.create-pr.outputs.pull_request_id}}
+#     BRANCH: ${{ steps.create-pr.outputs.temp_branch}}
+
+inputs:
+  files:
+    description: "Files to add separated by space"
+    required: true
+  temp_branch_prefix:
+    description: "Prefix for temporary git branch to be created, e.g, ci-docs"
+    required: true
+  pull_request_title:
+    description: "Pull Request title to use"
```

Comment:
This could be generated: "chore(ci): Bump version from X to Y"

Reply:
This is designed to be a reusable action, that's why many of these are designed to be inputs defined by the caller.

For now (aka this PR) we are using the action only as part of the versioning process, so hardcoding the PR message makes sense.

In a future PR we will use this same action to open a PR that updates the documentation with the new Lambda Layer version (for example). 

With this in mind, allowing the caller to define these fields makes sense.